Source/DataTypes/SvgPointCollection.cs

```diff
@@ -60,10 +60,12 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
         {
             if (value is string)
             {
-                var parser = new CoordinateParser(((string)value).Trim());
+                var coords = ((string) value).AsSpan().Trim();
+                var parser = new CoordinateParser();
```

Comment:
Is this Method often called, when yes the parser Could be an instance variable and if it is null on the First access initialized

Reply:
Made changes as suggested.